### PR TITLE
Blog post for the 2.0 release

### PR DESCRIPTION
**Pull request by IAlibay**
Here is an initial draft of the 2.0.0 release.

@MDAnalysis/coredevs please review.

### CODE REVIEW

File: _posts/2021-08-22-release-2.0.0.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+ - @fiszczyp
+
+MDAnalysis thanks [NumFOCUS][]'s continued support as the organisation's fiscale sponsor.
+
+- The MDAnalysis Team
+
+[roadmap]: {{ site.baseurl }}{% post_url 2019-11-06-roadmap %}
+[conda-forge channel]: https://anaconda.org/conda-forge/mdanalysis
+[PyPi]: https://pypi.org/project/MDAnalysis/
+[installation instructions in the User Guide]: https://userguide.mdanalysis.org/stable/installation.html
```
Comment (IAlibay) [inline _posts/2021-08-22-release-2.0.0.md:131]: I realise we've been neglecting our userguide responsibilities in favour of the main repo. I'm switching to stable from 2.0.0 for now. I assume we can release this and then work on bumping up the user guide afterwards.

Does that work for you @lilyminium ?

File: _posts/2021-08-22-release-2.0.0.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+For more help with installation see the [installation instructions in the User Guide][].
+
+## Notable new additions
+
+There were many awesome contributions to MDAnalysis for version 2.0.0. Here we list a few notable example of these. For more information please see the [CHANGELOG][].
+
+### RDKit converter
+
+[Google Summer of Code][] student @cbouy implemented a new [RDKit converter][] which allows for seemless conversions from RDKit to MDAnalysis structures.
```
Comment (fiona-naughton) [inline _posts/2021-08-22-release-2.0.0.md:34]: seemless --> seamless

File: _posts/2021-08-22-release-2.0.0.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+**NSF REU** student @edisj implemented both a [reader and writer for the H5MD format][].
+
+### Bond-Angle-Torsion coordinates converter
+
+@daveminh implemented a converter from Cartesian to [Bond-Angle-Torsion coordinates][].
+
+
+## Important fixes
+
+For a full list of bugfixes see the [CHANGELOG][]. The follow are selected fixes that could either lead to wrong results.
```
Comment (fiona-naughton) [inline _posts/2021-08-22-release-2.0.0.md:65]: follow --> following.

Maybe reword the second sentence since it kinda implies it's the fixes that might lead to the wrong results (e.g. change to 'fixes for bugs that could...')?. The 'either' is also extraneous here

File: _posts/2021-08-22-release-2.0.0.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+- Fixes an issue where `select_atom`, AtomGroup.unique, ResidueGroup.unique, and SegmentGroup.unique did not sort the output atoms (Issues [#3364][] [#2977][])
+- Fixes the sometimes wrong sorting of atoms into fragments when unwrapping (Issue [#3352][])
+- AtomGroup.center now works correctly for compounds + unwrapping (Issue [#2984][])
+
+### File formats
+
+- Fixes support for DL_POLY HISTORY files that contain cell information even if there are no periodic boundary conditions (Issue [#3314][])
+- PDBWriter will use chainID instead of segID (Issue [#3144][])
+- PDBParser and PDBWriter now assign and use the element attribute (Issues [#3030][] [#2422][])
```
Comment (fiona-naughton) [inline _posts/2021-08-22-release-2.0.0.md:77]: Issue 3030 seems to be about adding .nc files - from a quick glance at the CHANGELOG, maybe this was meant to be 2423?

File: _posts/2021-08-22-release-2.0.0.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+### Analyses
+
+- Fixes issue when atttempting to use/pass mean positions to PCA analysis (Issue [#2728][])
```
Comment (fiona-naughton) [inline _posts/2021-08-22-release-2.0.0.md:81]: atttempting --> attempting

File: _posts/2021-08-22-release-2.0.0.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+- Fixes issue with WaterBridgeAnalysis double counting waters (Issue [#3119][])
+- Documents and fixes the `density` keyword for `rdf.InterRDF_s` (Isuse [#2811][])
```
Comment (fiona-naughton) [inline _posts/2021-08-22-release-2.0.0.md:83]: Isuse --> issue

File: _posts/2021-08-22-release-2.0.0.md [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+- Fixed Janin analysis residue filtering, including CYSH (Issue [#2898][])
+
+## Changes to functionality
+- Deprecated `hbonds.hbond_analysis` has been removed in favour of `hydrogenbonds.hbond_analysis` (Issues [#2739][], [#2746][])
+- TPRParser now loads TPR files with `tpr_resid_from_one=True` by deafult, which starts TPR resid indexing from 1 (instead of 0 as in previous MDAnalysis versions) (Issue [#2364][], PR [#3152][])
```
Comment (fiona-naughton) [inline _posts/2021-08-22-release-2.0.0.md:88]: deafult --> default

File: _posts/2021-08-22-release-2.0.0.md [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+ - @hp115
+ - @ojeda-e
+ - @thadanipaarth
+ - @HenryKobin
+ - @1ut
+ - @sulays
+ - @ahy3nz
+ - @fiszczyp
+
+MDAnalysis thanks [NumFOCUS][]'s continued support as the organisation's fiscale sponsor.
```
Comment (fiona-naughton) [inline _posts/2021-08-22-release-2.0.0.md:124]: fiscale --> fiscal

**Review by fiona-naughton — CHANGES_REQUESTED** [review]
Looks good to me :) - mostly just some spelling fixes

File: _posts/2021-08-22-release-2.0.0.md (hunk 4 above)
Comment (IAlibay) [reply _posts/2021-08-22-release-2.0.0.md]: Great catch :) Changed it to 2423

File: _posts/2021-08-22-release-2.0.0.md (hunk 3 above)
Comment (IAlibay) [reply _posts/2021-08-22-release-2.0.0.md]: Re-written, what do you think?

File: _posts/2021-08-22-release-2.0.0.md [hunk 9]
```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: Version 1
```
Comment (orbeckst) [inline _posts/2021-08-22-release-2.0.0.md:3]:
```suggestion
title: MDAnalysis 2.0 is here
```

File: _posts/2021-08-22-release-2.0.0.md [hunk 10]
```diff
@@ -0,0 +1,162 @@
...
+---
+
+We are happy to release version 2.0.0 of MDAnalysis!
+
+This is a new major version, includes major API changes and concludes our current [roadmap][].
```
Comment (orbeckst) [inline _posts/2021-08-22-release-2.0.0.md:8]: ... now we need a new one :-)

File: _posts/2021-08-22-release-2.0.0.md [hunk 11]
```diff
@@ -0,0 +1,162 @@
...
+
+This version supports Python 3.6 through to 3.9 on Windows (32- and 64-bit), MacOS, and Linux. It also offers preliminary support for ppc64le and ARM64, but does not currently support Apple M1 processors.
```
Comment (orbeckst) [inline _posts/2021-08-22-release-2.0.0.md:10]: Support for legacy versions of Python is only available in MDAnalysis 1.x.

Comment (orbeckst) [reply _posts/2021-08-22-release-2.0.0.md:10]: It also contains a number of breaking changes in addition to a large number of updates and enhancements.

File: _posts/2021-08-22-release-2.0.0.md [hunk 12]
```diff
@@ -0,0 +1,162 @@
...
+
+[Google Summer of Code][] student @yuxuanzhuang implemented a means to serialize MDAnalysis.Universe objects, paving the way for [parallel analysis][].
+
+### Mean Square Displacement
+
+@hmacdope implemented a new [Mean Square Displacement][] analysis class.
+
+### Results class
+
+@PicoCentauri implemented a [Results class][] to store results from analysis classes.
```
Comment (orbeckst) [inline _posts/2021-08-22-release-2.0.0.md:52]: This change streamlines the API for analysis classes and paves the way for a soon-to-be release command line interface for MDAnalysis.

File: _posts/2021-08-22-release-2.0.0.md [hunk 13]
```diff
@@ -0,0 +1,162 @@
...
+
+### H5MD support
+
+**NSF REU** student @edisj implemented both a [reader and writer for the H5MD format][].
+
+### Bond-Angle-Torsion coordinates converter
```
Comment (orbeckst) [inline _posts/2021-08-22-release-2.0.0.md:58]: I wouldn't call BAT a converter because this muddles the waters. Let's only call "converter" the things in our converter framework. Perhaps just "analysis"?

File: _posts/2021-08-22-release-2.0.0.md [hunk 14]
```diff
@@ -0,0 +1,162 @@
...
+[PyPi]: https://pypi.org/project/MDAnalysis/
+[installation instructions in the User Guide]: https://userguide.mdanalysis.org/stable/installation.html
+[CHANGELOG]: https://github.com/MDAnalysis/mdanalysis/blob/release-2.0.0/package/CHANGELOG
+[Google Summer of Code]: https://summerofcode.withgoogle.com/
+[RDKit converter]: https://docs.mdanalysis.org/2.0.0/documentation_pages/converters/RDKit.html
+[ChEMBL27]: https://ftp.ebi.ac.uk/pub/databases/chembl/ChEMBLdb/releases/chembl_27/
+[OpenMM converter]: https://docs.mdanalysis.org/2.0.0/documentation_pages/converters/OpenMM.html
+[parallel analysis]: https://userguide.mdanalysis.org/stable/examples/analysis/custom_parallel_analysis.html
+[Mean Square Displacement]: https://docs.mdanalysis.org/2.0.0/documentation_pages/analysis/msd.html
+[Results class]: https://docs.mdanalysis.org/2.0.0/documentation_pages/analysis/base.html?highlight=results#MDAnalysis.analysis.base.AnalysisBase.results
```
Comment (orbeckst) [inline _posts/2021-08-22-release-2.0.0.md:139]: remove highlight?

File: _posts/2021-08-22-release-2.0.0.md [hunk 15]
```diff
@@ -0,0 +1,162 @@
...
+
+It currently accurately converts 90% of chemical space (based on [ChEMBL27][]), but future changes will improve this.
+
+### OpenMM converter
+
+@ahy3nz implemented an [OpenMM converter][] which can read OpenMM objects directly into MDAnalysis.
+
+### Universe serialization
+
+[Google Summer of Code][] student @yuxuanzhuang implemented a means to serialize MDAnalysis.Universe objects, paving the way for [parallel analysis][].
```
Comment (orbeckst) [inline _posts/2021-08-22-release-2.0.0.md:44]: link to https://www.mdanalysis.org/2020/08/31/final-report-yuxuan/ ? Something like
```
[serialize MDAnalysis.Universe objects]({{ site.baseurl }}{% post_url 2020-08-31-final-report-yuxuan %})
```
... but double check.

File: _posts/2021-08-22-release-2.0.0.md [hunk 16]
```diff
@@ -0,0 +1,162 @@
...
+
+For more help with installation see the [installation instructions in the User Guide][].
+
+## Notable new additions
+
+There were many awesome contributions to MDAnalysis for version 2.0.0. Here we list a few notable example of these. For more information please see the [CHANGELOG][].
+
+### RDKit converter
+
+[Google Summer of Code][] student @cbouy implemented a new [RDKit converter][] which allows for seamless conversions from RDKit to MDAnalysis structures.
```
Comment (orbeckst) [inline _posts/2021-08-22-release-2.0.0.md:34]: Would also link to the blog post
```
[blog post]({{ site.baseurl }}{% post_url 2020-08-29-gsoc-report-cbouy %})
```

**Review by orbeckst — CHANGES_REQUESTED** [review]
The title needs to be changed, the rest is suggestions.

THANK YOU!!!

File: _posts/2021-08-22-release-2.0.0.md (hunk 13 above)
Comment (IAlibay) [reply _posts/2021-08-22-release-2.0.0.md:58]: I'm not convinced we should call it an "analysis" method to be honest - I should raise an issue about this at some point, but I think it really should have been an MDAnalysis.lib component.

File: _posts/2021-08-22-release-2.0.0.md (hunk 14 above)
Comment (IAlibay) [reply _posts/2021-08-22-release-2.0.0.md:139]: updated to point at the actual Results class docs

**Comment by IAlibay** [comment]
That should be everything addressed.

**Review by orbeckst — APPROVED** [review]

File: _posts/2021-08-22-release-2.0.0.md (hunk 13 above)
Comment (orbeckst) [reply _posts/2021-08-22-release-2.0.0.md]: fair enough, something for the future and 3.0; I am happy with your change

**Comment by IAlibay** [comment]
Right everything is up on conda-forge, shall we merge?

File: _posts/2021-08-22-release-2.0.0.md (hunk 1 above)
Comment (lilyminium) [reply _posts/2021-08-22-release-2.0.0.md:133]: Yeah, I've been pretty neglectful as well. There's a handful of issues to address for 2.0, not hard, just need a little bit of sitting down and writing. We can get that out fast.

File: _posts/2021-08-22-release-2.0.0.md [hunk 17]
```diff
@@ -0,0 +1,166 @@
...
+## Deprecations
+- The `bfactors` attribute is now aliased to `tempfactors` and will be removed in 3.0.0 (Issue [#1901][])
+- `WaterBridgeAnalysis.generate_table()` now returns table information, with the `table` attribute being deprecated
+- Various analysis result attributes which are now stored in `Results` will be deprecated in 3.0.0 (Issue [#3261][])
+- In 3.0.0 the ParmEd classes will only be accessible from the `MDAnalysis.converters` module
+- In 2.1.0 the TRZReader will default to a dt of 1.0 ps when failing to read it from the input TRZ trajectory
+
+## Author statistics
+
+Altogether, this represents the work of 35 authors. 17 of which were new contributors:
```
Comment (lilyminium) [inline _posts/2021-08-22-release-2.0.0.md:106]:
```suggestion
Altogether, this represents the work of 35 authors, 17 of which were new contributors:
```

**Review by lilyminium — APPROVED** [review]
Thanks for putting this together!

File: _posts/2021-08-22-release-2.0.0.md (hunk 1 above)
Comment (IAlibay) [reply _posts/2021-08-22-release-2.0.0.md:133]: Please do ping us to do stuff, it's a collective responsibility of all coredevs to make sure we have a functioning userguide.
